contracts/Siphon.sol

```diff
@@ -7,22 +7,28 @@ import "./MultisendEncoder.sol";
 import "./IDebtPosition.sol";
 import "./ILiquidityPosition.sol";
 
+struct PaymentChannel {
+    address dp;
+    address lp;
+}
+
 contract Siphon is Module, MultisendEncoder {
-    mapping(address => bool) public dps;
-    mapping(address => bool) public lps;
+    mapping(string => PaymentChannel) public channels;
```

Comment:
Is there a reason you chose `string` rather than something like `uint16` (or even `uint8`, probably don't need more than 256 channels)?

String is nice because it's more human readable, I guess.